persistence/build.gradle.kts

```
@@ -0,0 +1,62 @@
+plugins {
+    Plugins.androidLibrary(this)
+    Plugins.multiplatform(this)
+}
+
+group = "com.wire.kalium"
+version = "0.0.1-SNAPSHOT"
+
+android {
+    compileSdk = Android.Sdk.compile
+    sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
+    defaultConfig {
+        minSdk = Android.Sdk.min
+        targetSdk = Android.Sdk.target
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
```

Shouldn't we also extract this to a common file inside the root `buildSrc` folder?

---

i didn't really get what you mean, this is the gradle to build the `persistence` module

---

I meant to centralise the java version of all the module `build.gradle.kts` files into one variable inside the shared `Dependencies` file
```suggestion
        sourceCompatibility = Versions.javaVersion
        targetCompatibility = Versions.javaVersion
```

---

it's done this way in all kalium modules, it make sense to be changed and unified across the app, so i will add a ticket and will change it in a later PR (i have enough changes in this one)